src/actions/assessment.js

```diff
@@ -56,13 +56,20 @@ export function updateAssessment(assessment, criteriaId){
               } else {
                 return criteria
               }
+            }).sort((a , b) => {
+              return a.id - b.id;
```

Comment:
now that you have the same fn 3 times, might make sense to make it a separate fn?